src/wechaty_puppet_itchat/puppet.py

```diff
@@ -1130,36 +1072,12 @@ async def on_message_group_file(msg):
                 timestamp=msg['CreateTime'],
                 to_id=msg['ToUserName']
             )
-
             self._event_stream.emit('message', event_message_payload)
-            await asyncio.sleep(0.1)
 
-        @itchat.msg_register(FRIENDS)
-        def add_friend(msg):
-            self.message_container[msg['MsgId']] = msg
-            log.info(f'receive file message info <{msg.user}>')
-            msg.user.verify()
-
-        # @itchat.msg_register(SYSTEM)
-        # def handle_system_msg(msg):
-        #     log.info(f'receive system message info <{msg["Text"]}>')
-
-        message_container = self.message_container.copy()
-
-        async def run(self, event_stream, payload):
-            async def reply_fn():
-                try:
-                    while self.alive:
-                        await self.configured_reply(event_stream=event_stream, payload=payload,
-                                                    message_container=message_container)
-                except KeyboardInterrupt:
-                    if self.useHotReload:
-                        await self.dump_login_status()
-                    self.alive = False
-
-            while True:
-                await asyncio.sleep(0.5)
-                await reply_fn()
-
-        self.itchat.run = types.MethodType(run, self.itchat.originInstance)
-        await self.itchat.run(event_stream=self._event_stream, payload=EventMessagePayload)
+        while True:
+            print('tick ...')
+            await asyncio.sleep(0.1)
```

Comment:
the code `asyncio.sleep(0.1)` is to suspend the current task, allowing other tasks to run. The official code is `asyncio.sleep()` . Please refer to: https://docs.python.org/3/library/asyncio-task.html#asyncio.sleep

Reply:
> ping @lyleshaw

I'm so sorry that it took me a long time to handle your review due to some delays